language-culture-and-medicine.html

```diff
@@ -7,8 +7,20 @@ <h1>Language Culture and Medicine</h1>
     <p><strong>How is English Now! Unique?</strong> English Now! has served students from the NIH medical research and related health care communities from its location in downtown Bethesda for decades. Our small class size makes us uniquely focused on personalization of curriculum and teaching to the specific needs of students. Our purpose to inspire students to engage, enjoy, and thrive, in their studies and during their time in the U.S. is aligned with the needs of high-level professional students. We do more than teach English: we provide the support you need to communicate successfully in the global workplace. Also, ask us about our Beyond the Classroom programs: office hours with our staff, monthly lunchtime talks, potluck lunches, and outings.</p>
     <p><strong>Course Schedule, Tuition &amp; Registration:</strong> (1) We teach this class as a semi-private lesson for two or three students with similar needs, levels, and schedules. Do you have a colleague that might be interested? For a semi-private lesson, tuition is $672 for 16 sessions. (2) This class is also offered as a private lesson, at dates and times at your convenience. Private lessons, tuition is $43 per class hour, plus registration fee. All students pay a $40, one-time registration fee. Books and materials are extra and usually between $30 and $40.</p>
+    <div>We also offer graduate-level classes at National Institutes of Health in collaboration with the Foundation for Advanced Education in the Sciences (FAES). Students can earn three credits for completion of these courses. Classes for Fall 2019 are as follows:</div>
+    <p>
+        <div>Advanced Conversation: Language, Culture and Medicine</div>
+        <div>Schedule: September 10 - December 17, Tuesdays and Thursdays, 4:00 p.m.-5:30 p.m.</div>
+        <div>Tuition: $480</div>
+    </p>
+    <p>
+        <div>English for Workplace Communication</div>
+        <div>Schedule: September 10 - December 17, Tuesdays and Thursdays, 1:00 p.m. - 2:30 p.m.</div>
+        <div>Tuition: $480</div>
+    </p>
+    <div>You can register for Advanced Conversation: Language, Culture and Medicine (English 355) or English for Workplace Communication (Engish 375) online at <a href="https://my.faes.org/Common/CourseSchedule.aspx" target="_blank">https://my.faes.org/Common/CourseSchedule.aspx</a>. Please contact Jennifer Kagan, English Now! Director of Program Development and English Language Instructor, and FAES Faculty, at <a href onclick="emailMe('jennifer.kagan')">jennifer.kagan@english-now.com</a>, with any questions about these courses.</div>
```

Comment:
Why `target=_blank`?

Reply:
I just thought it would be useful to open in a new tab because it takes the user to another site.

Reply:
I would remove this because we don't do it consistently throughout the website.